### ISSUE TITLE
tones

### ISSUE THREAD

**Issue by MacyL** [lexibank/lexitools#9]
information about tones are listing in the etc/ folder, and design a function to take the information and merge into a wordlist output.

**Comment by LinguList** [comment]
Yes, the "simpler" tool would be (for starters): provide a list of datasets and language names, and create a wordlist TSV file. Then, adding tones could be an option. 

In fact, we should start by adding some functions to check the overlap in concepts.

**Comment by LinguList** [comment]
so you could say: `lexitools overlap chehmongmien sagartst`

and it outputs a list of concepticon glosses.

**Comment by LinguList** [comment]
then we would have `lexitools wordlist chenhmongmien sagartst --languages=A,B,C,D --concepts=A,B,C,D`

**Comment by MacyL** [comment]
this is good, but just a question : `concepticon intersect` can also check the overlapping?

**Comment by LinguList** [comment]
yes, you are right, so start with the merging :)

**Comment by MacyL** [comment]
cool :)

**Comment by MacyL** [comment]
[NOTE] use Chen and White Hmong in WOLD as test dataset.
